### PR TITLE
Generate release note

### PR DESCRIPTION
**Pull request by daisy-ycguo**
This PR is to generate a release note.

It reads a template, append change logs and create a *.md file in the folder of `$WORK_DIR`.
Try it out by:
```
./tools/generate_change_logs.sh <WORK_DIR>
```

### CODE REVIEW

File: tools/download_source_code.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ function git_clone_repo() {
     git clone --depth $clone_depth -b $BRANCH $REPO $OPENWHISK_SOURCE_DIR/$PROJECT_NAME
     cd $OPENWHISK_SOURCE_DIR/$PROJECT_NAME
     git reset --hard $HASH
-    rm -rf .git
+    #rm -rf .git
```
Comment (houshengbo) [inline tools/download_source_code.sh]: rm -rf .git can stay here.

File: tools/package_source_code.sh [hunk 2]
```diff
@@ -33,5 +33,7 @@ rsync -rtp --exclude .*\* --exclude bin\* --exclude build\* $OPENWHISK_SOURCE_DI
 for repo in $(echo $repos | sed "s/,/ /g")
 do
     repo_name=$(echo "$repo" | sed -e 's/^"//' -e 's/"$//')
-    cd $OPENWHISK_CLEANED_SOURCE_DIR/$repo_name && tar czf ${CURRENT_VERSION_DIR}/${repo_name}-${version}-sources.tar.gz .
+    cd $OPENWHISK_CLEANED_SOURCE_DIR/$repo_name
```
Comment (houshengbo) [inline tools/package_source_code.sh]: no need to change this file.

**Comment by daisy-ycguo** [comment]
@houshengbo `rm -rf .git` will remove all the git information and `git log` will report error. I do need to keep `.git` folder and remove it just before packaging.

**Comment by houshengbo** [comment]
Cool, I see the point of moving the command "rm -rf .git".

**Pull request merged**